src/test/perl/tabcompletion.t

```diff
@@ -25,12 +25,12 @@ $mock_cli->mock('_get_uid', 0);
 # Do not mock the renderer, use the one from CCM
 # try source relative to current path (eg on jenkins) and default install
 my $tt_ccm = abs_path(getcwd().'/../CCM/src/main/resources');
-ok(-d $tt_ccm, "Did not find the CCM TT directory $tt_ccm");
+ok(-d $tt_ccm, "Did not find the CCM TT directory $tt_ccm") if defined $tt_ccm;
```

Comment:
Ah, so the tests need CCM to be checked out somewhere nearby?

Reply:
during the initial setup, you actually install the CCM rpm, so you should fallback to the rpm path before giving up